eks-worker-al2.json

```
@@ -14,9 +14,9 @@
     "binary_bucket_region": "us-west-2",
     "kubernetes_version": null,
     "kubernetes_build_date": null,
-    "docker_version": "19.03.6ce-1.amzn2",
+    "docker_version": "18.09.9ce-2.amzn2",
```

why are we downgrading docker


 it should  be this 
```suggestion
    "docker_version": "19.03.6ce-2.amzn2",
```